.doks/Makefile

```
@@ -33,6 +34,7 @@ deploy_to_doks_from_chartmuseum: set_cluster_context
 		--namespace ${NAMESPACE} \
 		--set global.imagePullSecrets[0].name=docker-credentials \
 		--set global.play.cryptoSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
+		--set global.akka.sessionSecret=$${SHARED_AKKA_SESSION_SECRET} \
```

```suggestion
		--set global.akka.sessionSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
```

Since this is a dev environment I think we can use the same "secret" fall everything

---

I added the env to the circle ci pipeline